cumulus_library/studies/discovery/code_detection.sql

```diff
@@ -0,0 +1,41 @@
+-- noqa: disable=all
+/*
+This is a reference output of the SQL generated by builder_code_detection.py 
+that is used by the core__encounter_type table. It is not invoked directly.
```

Comment:
This makes me wonder: why is it checked in? Does anything reference it? How is it generated? Can it be out of date?

This sql is also in the unit test as an expected string, yeah?

Reply:
Yeah but maybe comment some of that stuff - like why it exists. How to reproduce it - maybe a comment in the unit test that also uses this same SQL like "if you change this unit test, also go regenerate our sample in this location" -- all that sort of stuff. Or better yet, use this SQL directly in the unit test so it won't be possible to get stale.

Should this file be in the shipped python or in the test code?

All these questions swirl in my mind when I see a pile of SQL that might bitrot in a week. I'd vote for some more comments/safeguards etc.

Reply:
Ok, I've updated this to reflect the synthea format, for the time being.

Per our side conversation, I created a ticket to get this into regression against the synthea dataset: #135